csds-client/README.md

```diff
@@ -0,0 +1,48 @@
+# CSDS Client
+[Client status discovery service (CSDS)](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/status/v2/csds.proto) is a generic xDS API that can be used to get information about data plane clients from the control plane’s point of view. It is useful to enhance debuggability of the service mesh, where lots of xDS clients are connected to the control plane.<br/>
```

Comment:
nit: link to v3 proto?

Reply:
Not sure for this. Since for now this client only supports v2, I linked this with v2 proto. Will v3 proto be better for here?

Reply:
in general, use the latest xds API, as v2 is considered deprecated. 

BTW, we should add v3 support in the next PR.

Reply:
Solved.